BraintreeCore/src/main/java/com/braintreepayments/api/core/LinkType.kt

```diff
@@ -0,0 +1,11 @@
+package com.braintreepayments.api.core
+
+/**
+ * Used to describe the link type for analytics
+ * Note: This enum is exposed for internal Braintree use only. Do not use.
+ * It is not covered by Semantic Versioning and may change or be removed at any time.
+ */
+public enum class LinkType(val stringValue: String) {
+    UNIVERSAL("universal"),
```

Comment:
On Android, the other type of link is App Links. Not sure if we want to keep the iOS nomenclature for analytics or use Android specific one.

Reply:
I see that there was some discussion around this. Please dis-regard if the rest of the team has come to a conclusion.